conf/defaults.yaml

```diff
@@ -231,16 +228,13 @@ topology.multilang.serializer: "org.apache.storm.multilang.JsonSerializer"
 topology.shellbolt.max.pending: 100
 topology.skip.missing.kryo.registrations: false
 topology.max.task.parallelism: null
-topology.max.spout.pending: null
+topology.max.spout.pending: null  # TODO: We dont need this any more
 topology.state.synchronization.timeout.secs: 60
-topology.stats.sample.rate: 0.05
+topology.stats.sample.rate: 0.001
```

Comment:
Sampling 5 out of every 100 seemed bit excessive even for topos doing several thousand per second. Does 1 out of every 100 seem more reasonable ?